### ISSUE TITLE
Add default og:image to blogposts

### ISSUE THREAD

**Issue by werthen** [ZeusWPI/zeus.ugent.be#248]
There's no image on our blogposts when posted on facebook. Adding an og:image attribute (pointing to the photo on our facebook page) would make this look way prettier.

**Comment by rien** [comment]
Maybe this could be combined with the option to add an image banner to a blogpost?

**Comment by werthen** [comment]
Yes, but the default image shouldn't be a banner. However, when a banner is supplied, we could use that image for the facebook preview

**Comment by rien** [comment]
Then I suggest adding a postprocessing step to find the first image tag and take that URL.

If no images are included we could just use the Zeus WPI logo.
